src/main/java/com/shopify/model/ShopifyCustomer.java

```diff
@@ -1,22 +1,19 @@
 package com.shopify.model;
 
-import java.math.BigDecimal;
+import com.shopify.model.adapters.DateTimeAdapter;
+import org.joda.time.DateTime;
 
 import javax.xml.bind.annotation.XmlAccessType;
 import javax.xml.bind.annotation.XmlAccessorType;
 import javax.xml.bind.annotation.XmlElement;
 import javax.xml.bind.annotation.XmlRootElement;
 import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
 
-import org.joda.time.DateTime;
-
-import com.shopify.model.adapters.DateTimeAdapter;
-
 @XmlRootElement
 @XmlAccessorType(XmlAccessType.FIELD)
 public class ShopifyCustomer {
 
-	private String id;
+	private Long id;
```

Comment:
Is there a reason specifically that you need this to be a long? The only thing concerning to me is that this is a breaking change to existing functionality and could require a lot of changes for anyone using this, It also becomes inconsistent with the rest of the shopify sdk here that stuff like orders is still using a string. Don't disagree with the change, I just think if we are going to change it for customer, we should try to handle off of the pl aces that an ID is a string when it should be a number.

Reply:
I was getting errors from the Shopify API when it was sent over as a String. I can give details of the error shortly. 

Re: string vs long ids elsewhere I agree in principle. This is the challenge of working with a statically-typed language against a ruby dynamic-typed language-based API.